crates/vfio-ioctls/src/vfio_device.rs

```diff
@@ -243,7 +237,7 @@ impl VfioContainer {
         }
 
         // Add the new group object to the hypervisor driver.
-        #[cfg(any(feature = "kvm", all(feature = "mshv", target_arch = "x86_64")))]
+        #[cfg(any(feature = "kvm", feature = "mshv"))]
```

Comment:
Can this be dropped all together?

Reply:
Makes sense. Dropped that check.

Reply:
That seems to be a trouble when compiling with no-default-features. So re-introducing the check.